packages/flutter_tools/lib/src/migrations/lldb_init_migration.dart

```diff
@@ -0,0 +1,285 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:xml/xml.dart';
+import 'package:xml/xpath.dart';
+
+import '../base/file_system.dart';
+import '../base/project_migrator.dart';
+import '../build_info.dart';
+import '../ios/xcodeproj.dart';
+import '../project.dart';
+
+class LLDBInitMigration extends ProjectMigrator {
```

Comment:
Should we update the app template's scheme as well for new projects?

`packages/flutter_tools/templates/app/ios-swift.tmpl/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme.tmpl`?